parm/config/config.base.emc.dyn

```diff
@@ -179,21 +179,20 @@ case "${APP}" in
     export DO_ICE="YES"
     export CCPP_SUITE="FV3_GFS_v16_coupled_nsstNoahmpUGWPv1"
     export WAVE_CDUMP="both"
-    export confignamevarfornems="cpld_wave"
     export cplwav2atm=".true."
+    export confignamevarfornems="cpld_wave"
     source $EXPDIR/config.defaults.s2sw
     ;;
-  ATMAERO)
-    export DO_COUPLED="YES"
-    export DO_AERO="YES"
-    export confignamevarfornems="atm_aer"
-    ;;
   *)
     echo "Unrecognized APP: ${1}"
     exit 1
     ;;
 esac
 
+if [[ $DO_AERO = "YES" ]]; then
+    export confignamevarfornems="${confignamevarfornems}_aero"
```

Comment:
Is this the naming convention we want to use?   I'm working on updates to ufs-weather-model to add this to the regression tests and this is not the naming convention I used: https://github.com/JessicaMeixner-NOAA/ufs-weather-model/blob/feature/p8b_aero/tests/parm/nems.configure.cpld_aero_wave.IN I'm fine with either (I went alphabetical aero_wave instead of wave_aero), but we should be consistent between the two to make things simpler.  So we should either change that here or change that in my branch.

Reply:
Okay, but this would be a 3 line change to change this here and then renaming the nems.congifure file versus having to change this in 20+ different test descriptions in rt.sh + renaming all the files in my branch of the workflow for p8 plus the ufs- branch.   So if logically this is a better reason, okay, but it's non-trivial to change this on my end so I want to make sure this is really what we want to do before I spend the time changing it on my end.

Reply:
We can certainly modify the template names and add a test for waves following the aerosol one:
```
if [[ $DO_WAVE = "YES" ]]; then
  export confignamevarfornems="${confignamevarfornems}_wave
fi
```

Ideally, this could be streamlined as follows:
```
for comp in ${COMPONENTS} ; do
  export confignamevarfornems="${confignamevarfornems}_${comp}
done
```
where `COMPONENTS` is an ordered list of lowercase component names. This construct can be extended to other portions of the global workflow, also using `case ` if needed:
```
for comp in ${COMPONENTS} ; do
  case ${comp} in
    aero)
      ... process aerosol options ...
      ;;
    wave)
      ... process wave options ...
      ;;
    ...
  esac
done
```

Reply:
@WalterKolczynski-NOAA any thoughts on this?  I need to know so if I have to change my ufs-waether-model work I can start that ASAP as there will be a lot to change and test.

Reply:
Let's go ahead with your convention. I'll change the PR.

Reply:
Thanks Walter - let me know if you need anything from me, I have some of these updates here as well, if that's helpful: https://github.com/jessicameixner-noaa/global-workflow/tree/feature/p8b. Let me know if there's anything I can do to help with this change.